Cargo.toml

```diff
@@ -26,18 +26,20 @@ plugin = false
 proc-macro = false
 
 [dependencies]
-lebe = "0.5.1"        # generic binary serialization
-half = "1.5.0"        # 16 bit float pixel data type
-bit_field = "0.10.0"  # exr file version bit flags
-libflate = "0.1.27"   # zlib compression
```

Comment:
That is a bit up for debate, some crates never reach 1.0. I normally use major.minor allowing for the build number to vary so that small fixes can be applied. But I will change to conform to your code style, it has bad from my part to change that.

Reply:
Yes, it's even already a meme that most crates never make it to 1.0, haha